website/content/commands/tls/ca.mdx

```diff
@@ -7,40 +7,45 @@ page_title: 'Commands: TLS CA Create'
 
 Command: `consul tls ca create`
 
-This command create a self signed CA to be used for Consul TLS setup.
+This command creates a self-signed CA to be used for Consul TLS setup.
 
 ## Example
 
-Create CA:
+Create a CA:
 
 ```shell-session
 $ consul tls ca create
 ==> Saved consul-ca.pem
 ==> Saved consul-ca-key.pem
 ```
 
+Create a CA which only signs certificates for example.com
+
+```shell-session
+$ consul tls ca create -name-constraint -domain example.com
+==> Saved example.com-ca.pem
+==> Saved example.com-ca-key.pem
+```
+
 ## Usage
 
-Usage: `consul tls ca create [filename-prefix] [options]`
+Usage: `consul tls ca create [options]`
 
 #### Command Options
 
 - `-additional-name-constraint=<value>` - Add name constraints for the CA.
   Results in rejecting certificates for other DNS than specified. Can be used
   multiple times. Only used in combination with `-name-constraint`.
 
-- `-days=<int>` - Provide number of days the CA is valid for from now on, defaults to 5 years.
+- `-days=<int>` - Provide number of days for which the CA will be valid. Defaults to 5 years.
 
-- `-domain=<string>` - Domain of consul cluster. Only used in combination with
-  `-name-constraint`. Defaults to `consul`.
+- `-domain=<string>` - Domain of consul cluster. Only used when `-name-constraint` is set. Defaults to consul.
+  Additional domains can be passed with `-additional-name-constraint`.
```

Comment:
To be honest, I'm not super clear on the terminology that should be used here because the "domain" of TLS CA and "domain" of Consul's DNS server overlap but aren't exactly the same.

How does this look @boruszak :

> The DNS domain of the Consul cluster that agents are [configured](https://developer.hashicorp.com/consul/docs/agent/config/cli-flags#_domain) with.
> Defaults to `consul`. Only used when `-name-constraint` is set.
> Additional domains can be passed with `-additional-name-constraint`.